src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -119,11 +129,13 @@ impl<S: StateStore> StorageTableInner<S, EitherSerde> {
     /// from those supplied to associated executors.
     /// These `output_column_ids` may have `pk` appended, since they will be needed to scan from
     /// storage. The associated executors may not have these `pk` fields.
-    pub fn new_partial(
+    pub fn new_partial_inner(
```

Comment:
I didn't get the purpose of adding this new method. Why do we need to exclude some columns? I think we shouldn't have an extra table catalog for subscription. It should use the table catalog of upstream mv directly, and then we won't have any extra column to be excluded.

Reply:
Yes, we are using the upstream catalog here, but we need to add an “op” column, which we don't have in the store, so we need to exclude it here to avoid some errors.